<!doctype html>
<html lang="en">
  <head>
     
    <title>PRECEPT: Publications</title>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css"
          integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
    <link href="assets/css/site.css" rel="stylesheet" />
      </head>

  <body>
      <div class="container"> 
      <!-- <div class="container-fluid"">  -->
         

      <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
            <a class="navbar-brand" href="index.html">PRECEPT</a>
            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
              <span class="navbar-toggler-icon"></span>
            </button>
            <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
              <div class="navbar-nav">

                                  <a class="nav-item nav-link"href="index.html">Home</a>
                                  <a class="nav-item nav-link"href="team.html">Team</a>
                                  <a class="nav-item nav-link active"href="publications.html">Publications</a>
                                  <a class="nav-item nav-link"href="presentations.html">Presentations</a>
                                  <a class="nav-item nav-link"href="funding.html">Funding</a>
                                  <a class="nav-item nav-link"href="contact.html">Contact</a>
                
              </div>
            </div>
          </nav>
      

<h1 class="display-4 my-5">Publications</h1>


  <h3 class="my-5">2019</h3>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Bionitio: demonstrating and facilitating best practices for bioinformatics command-line software</a>, GigaScience, 2019.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Peter Georgeson, Anna Syme, Clare Sloggett, Jessica Chung, Harriet Dashnow, Michael Milton, Andrew Lonsdale, David Powell, Torsten Seemann, Bernard Pope</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">GigaScience</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">8</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">9</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2019</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1093/gigascience/giz109</dd>
                  
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="https://doi.org/10.1093/gigascience/giz109">https://doi.org/10.1093/gigascience/giz109</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10"><p>Background. Bioinformatics software tools are often created ad hoc, frequently by people without extensive training in software development. In particular, for beginners, the barrier to entry in bioinformatics software development is high, especially if they want to adopt good programming practices. Even experienced developers do not always follow best practices. This results in the proliferation of poorer-quality bioinformatics software, leading to limited scalability and inefficient use of resources; lack of reproducibility, usability, adaptability, and interoperability; and erroneous or inaccurate results.</p><p>Findings. We have developed Bionitio, a tool that automates the process of starting new bioinformatics software projects following recommended best practices. With a single command, the user can create a new well-structured project in 1 of 12 programming languages. The resulting software is functional, carrying out a prototypical bioinformatics task, and thus serves as both a working example and a template for building new tools. Key features include command-line argument parsing, error handling, progress logging, defined exit status values, a test suite, a version number, standardized building and packaging, user documentation, code documentation, a standard open source software license, software revision control, and containerization.</p><p>Conclusions. Bionitio serves as a learning aid for beginner-to-intermediate bioinformatics programmers and provides an excellent starting point for new projects. This helps developers adopt good programming practices from the beginning of a project and encourages high-quality tools to be developed more rapidly. This also benefits users because tools are more easily installed and consistent in their usage. Bionitio is released as open source software under the MIT License and is available at <a href="https://github.com/bionitio-team/bionitio">https://github.com/bionitio-team/bionitio</a></p>
</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

  
  <h3 class="my-5">2018</h3>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>sEst: Accurate Sex-Estimation and Abnormality Detection in Methylation Microarray Data</a>, International Journal of Molecular Sciences, 2018.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Chol-Hee Jung, Daniel J. Park, Peter Georgeson, Khalid Mahmood, Roger L. Milne, Melissa C. Southey, Bernard J. Pope</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">International Journal of Molecular Sciences</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">19</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">10</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2018</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.3390/ijms19103172</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">30326623</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.3390/ijms19103172">http://dx.doi.org/10.3390/ijms19103172</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">DNA methylation, sex information, sex-chromosome abnormalities, epigenetics</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">DNA methylation influences predisposition, development and prognosis for many diseases, including cancer. However, it is not uncommon to encounter samples with incorrect sex labelling or atypical sex chromosome arrangement. Sex is one of the strongest influencers of the genomic distribution of DNA methylation and, therefore, correct assignment of sex and filtering of abnormal samples are essential for the quality control of study data. Differences in sex chromosome copy numbers between sexes and X-chromosome inactivation in females result in distinctive sex-specific patterns in the distribution of DNA methylation levels. In this study, we present a software tool, sEst, which incorporates clustering analysis to infer sex and to detect sex-chromosome abnormalities from DNA methylation microarray data. Testing with two publicly available datasets demonstrated that sEst not only correctly inferred the sex of the test samples, but also identified mislabelled samples and samples with potential sex-chromosome abnormalities, such as Klinefelter syndrome and Turner syndrome, the latter being a feature not offered by existing methods. Considering that sex and the sex-chromosome abnormalities can have large effects on many phenotypes, including diseases, our method can make a significant contribution to DNA methylation studies that are based on microarray platforms.
</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

        </div> <!-- container -->

      <!-- jQuery first, then Popper.js, then Bootstrap JS -->
      <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.3/umd/popper.min.js" integrity="sha384-vFJXuSJphROIrBnz7yo7oB41mKfc8JzQZiCq4NCceLEaO4IHwicKwpJf9c9IpFgh" crossorigin="anonymous"></script>
      <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/js/bootstrap.min.js" integrity="sha384-alpBpkh1PFOepccYVYDB4do5UnbKysX5WZXm3XxPqe5iKTfUKjNkCk9SaVuEZflJ" crossorigin="anonymous"></script>
      <script src="assets/js/site.js"></script>
      <script src="assets/js/details-element-polyfill.js"></script>
  </body>
</html>
